public/assets/js/scripts.js

```
@@ -0,0 +1,617 @@
+      me_data = me.data('confirm');
+
+  me_data = me_data.split("|");
+  me.fireModal({
+    title: me_data[0],
+    body: me_data[1],
+    buttons: [
+      {
+        text: me.data('confirm-text-yes') || 'Yes',
+        class: 'btn btn-danger btn-shadow',
+        handler: function() {
+          eval(me.data('confirm-yes'));
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)